Chinese/Orascom/Orascom.md

```diff
@@ -0,0 +1,12 @@
+曾拿下四年唯一运营权
+Orascom电信控股公司1998年成立。在世界各地的扩张始终围绕一个核心策略：就是获得新兴市场的先发优势。1999年落地邻国约旦。又一年之后，Orascom拓展到11个撒哈拉以南的非洲国家，2003年美伊战争爆发后进入伊拉克。其结果也十分显著，一跃成为中东、非洲、巴基斯坦三地规模最大、业务种类最全的网络运营商。
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.